### PR TITLE
String aug

### PR DESCRIPTION
**Pull request by christianvedels**
This PR implements
1. AttackerClass with almost the same functionality as TextAttack but without this as a dependency. This addressed issues #53 and #58
2. A bugfix for the AttackerClass. The inequality was reverted in `if random.random() > alt_prob`

3. `adversarial_occupations.py` which generates adversarial occupational descriptions. This is done in 4 separate ways:
     a. Occupational descriptions, `occ1`, are doubly translated to English and back to the original language until a goal function is fulfilled. (For English it is translated to French and back)
     b. Occupational descriptions, are altered using the AttackerClass.attack() until a goal function is fulfilled
     c. Occupational descriptions are translated from every language and to every language.
     d. Random gibberish is generated with the label '-1'. Random gibberish is by definition '-1'

**Augmentable strings:**
For a) and b) only *augmentable string* are considered. These fulfill two properties:
1. When they are translated they are not the same (this typically indicates something the translator does not pick up)
2. `OccCANINE.predict()` does not predict the correct label

**Goal function:**
We know that OccCANINE is initially able to predict all the correct labels. When this changes, this is a successful adversarial occupation, which is then returned and eventually stored. 

**Translation module:**
Translation is based on https://huggingface.co/facebook/m2m100_418M; https://arxiv.org/pdf/2010.11125

**Still to fix**
- The dataloader should load these examples in a balanced manner compared to the real training data
- Relative import of AttackerClass marked with FIXME

### CODE REVIEW

**Comment by TorbenSDJohansen** [comment]
Relative import issue fixed in #67, along with some minor improvements.
